<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4_CatsMaintWidget</class>
 <widget class="QWidget" name="Qt4_CatsMaintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>928</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,0,0,0,0,1">
     <item>
      <widget class="QGroupBox" name="boxPower">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Arm Power</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblPowerState">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btPowerOn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Power On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btPowerOff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Power Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxRegulation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>LN2 Regulation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblRegulationState">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Regulation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btRegulationOn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Regulation On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btRegulationOff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Regulation Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutLids" stretch="1,1,1">
       <item>
        <widget class="QGroupBox" name="boxLid1">
         <property name="title">
          <string>Lid 1</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="btLid1Open">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btLid1Close">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxLid2">
         <property name="title">
          <string>Lid 2</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="btLid2Open">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btLid2Close">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxLid3">
         <property name="title">
          <string>Lid 3</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="btLid3Open">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btLid3Close">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="boxTools">
       <property name="title">
        <string>Tools</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btOpenTool">
              <property name="text">
               <string>Open Tool</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btCloseTool">
              <property name="text">
               <string>Close Tool</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btToolCalib">
              <property name="text">
               <string>Tool Calib</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btMagnetOff">
              <property name="text">
               <string>Magnet on</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btMagnetOn">
              <property name="text">
               <string>Magnet Off</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxBarcode">
       <property name="title">
        <string>Barcode</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Barcode:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="barcodeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: white;
color: #009;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>........</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="barcodeCheck">
            <property name="text">
             <string>Read Barcode during trajectory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btBarcodeRead">
            <property name="text">
             <string>Read now</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxCommands">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Commands</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Actions:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btBack">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font/>
              </property>
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btSafe">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font/>
              </property>
              <property name="text">
               <string>Safe</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btDry">
              <property name="text">
               <string>Dry</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btHome">
              <property name="text">
               <string>Home</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btMore">
              <property name="text">
               <string>More...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Recovery:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btResetPutGet">
              <property name="text">
               <string>Reset put/get</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btRecoverFailure">
              <property name="text">
               <string>Recover Fail</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btResetMotion">
              <property name="text">
               <string>Reset Motion</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btMemClear">
              <property name="text">
               <string>Clear Memory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAbort">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox6">
       <property name="title">
        <string>CATS message</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="lblMessage">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Dummy message</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>btPowerOn</tabstop>
  <tabstop>btPowerOff</tabstop>
  <tabstop>btLid1Open</tabstop>
  <tabstop>btLid1Close</tabstop>
  <tabstop>btLid2Open</tabstop>
  <tabstop>btLid2Close</tabstop>
  <tabstop>btLid3Open</tabstop>
  <tabstop>btLid3Close</tabstop>
  <tabstop>btBack</tabstop>
  <tabstop>btSafe</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
